lib/routes/account.js

```diff
@@ -922,7 +966,9 @@ module.exports = function (
             // There's code in the handler that checks for a valid email,
             // no point adding overhead by doing it again here.
             email: isA.string().required(),
-            verified: isA.boolean().required()
+            verified: isA.boolean().required(),
+            sessionVerified: isA.boolean().required(),
```

Comment:
@vbudhram: I am filling out the deployment doc and thinking about deployment, and rollback. If /recovery_email/status is polled with a session that must be verified, but we then disable signin confirmation, I think we should use the old behavior and consider the session verified as long as the email is verified. I'm not sure about whether we should send the new fields either. This would allow us to soft roll back signin confirmation using config only, without rolling back any code.

@vbudhram, @philbooth, @rfk - thoughts?